src/ray/core_worker/transport/actor_task_submitter.cc

```diff
@@ -679,6 +679,18 @@ void ActorTaskSubmitter::HandlePushTaskReply(const Status &status,
   }
 }
 
+std::optional<rpc::ActorTableData::ActorState> ActorTaskSubmitter::GetActorState(
+    const ActorID &actor_id) const {
+  absl::MutexLock lock(&mu_);
+
+  auto iter = client_queues_.find(actor_id);
+  if (iter == client_queues_.end()) {
+    return std::nullopt;
+  } else {
+    return iter->second.state;
```

Comment:
Due to #24600, we currently can only get the latest actor state after the first actor task.